Packages/KeyAppKit/Sources/KeyAppNetwork/RequestMethod.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-enum RequestMethod: String {
+public enum RequestMethod: String {
```

Comment:
make it as nested enum of a HTTPClient, since it's part of it?

Reply:
Lets avoid nesting since we have already moved this to a separated packaged